### ISSUE TITLE
Start work on JS1 Skills Application session

### ISSUE THREAD

**Issue by illicitonion** [CodeYourFuture/syllabus#130]
**Which module(s) and week(s) does this change effect?**
Module(s): JS1
Week(s): 4

**What is the work that needs to be done?**

As per [the accepted proposal](https://docs.google.com/document/d/1OtFwkgXf4hJt8g_aePoRPRU-FBxWSbpC6Ym5O9taQm0/edit) we should start work on the skills application section for JS1

We should probably do https://github.com/CodeYourFuture/syllabus/issues/69 first or in conjunction with this task.

**Why is this work important to do?**

To help ensure our learners are well equipped to fully understand and apply the knowledge they've just learned.

**Comment by 40thieves** [comment]
While working on some exercises for JS2, I developed an idea with checking poker hands: https://repl.it/@40thieves/WhichPassionatePlanes#index.js (note: this is the solved version).

I discussed briefly with @illicitonion and we think it's a bit "too difficult" for a regular at-home coursework exercise. There is some out-of-the box thinking (e.g. finding duplicates to detect pairs). However, we think that it might be a good exercise if there was some volunteers available to provide prompts/hints. Therefore we think it might be helpful in the context of an application/consolidation workshop.
